outputs.tf

```
@@ -0,0 +1,4 @@
+output "deployment_id" {
+  description = "Print Deployment ID"
+  value =  var.deployment_id == "" ? random_string.deployment_id[0].id : var.deployment_id
+}
```

```suggestion
}

output "vpc_module_outputs" {
  description = "VPC outputs"
  value = module.vpc
}
```
This is how to print module outputs.